### ISSUE TITLE
Rotating doors don't work as expected

### ISSUE THREAD

**Issue by DerpedCrusader** [nzp-team/nzportable#908]
**Describe the bug**
instead of doors rotating along the origin axis, it seems to rotate in a skewed angle

**To Reproduce**
So, I'm providing .map/.bsp for both [Half-life](https://github.com/user-attachments/files/17087943/Half_life.zip) and [NZP](https://github.com/user-attachments/files/17087944/quake.zip)

I made a working door in Half life, and then converted it to, what should be, a working door in nzp, using a standard `func_door` with a spawn flag of `256`

1. open the above map in nzp
2. touch the door
3. door should open weirdly

**Expected behavior**
to open the same way it does in half life

**Screenshots/Footage**

Half Life:

https://github.com/user-attachments/assets/be701275-a631-4b1d-a6bb-0c87d73c37ab

NZP:
https://github.com/user-attachments/assets/f6a18073-8906-46c2-9a7f-dc13bd2d7d1f

**Affected Platforms**

-FTE
-Vita

PSP map loads, but hard crashes when you touch the door, with nothing in the error logs after map load

3DS crashes on map load

**Comment by MotoLegacy** [comment]
Can you test this on something other than FTE?

**Comment by DerpedCrusader** [comment]
updated

**Comment by MotoLegacy** [comment]
Were these bsps produced using the same map compiler?

**Comment by DerpedCrusader** [comment]
yes, using the one provided by us (i used a different one before, but its the same result from both)
